Dockerfile

```diff
@@ -31,3 +31,5 @@ RUN npm install pg
 RUN curl -SL https://jdbc.postgresql.org/download/postgresql-9.4.1207.jar > /postgres.jar
 
 RUN git clone https://github.com/cockroachdb/finagle-postgres.git && cd finagle-postgres && git checkout fatjartests && ./sbt assembly && mv target/scala-2.11/finagle-postgres-tests.jar /
+
+RUN curl -SL https://binaries.cockroachdb.com/cockroach-beta-20160407.linux-amd64.tgz | tar xvz
```

Comment:
Is there a better way to version this? I.e.

```
ENV COCKROACH_VERSION="beta-20160407"
...
RUN curl -SL https://binaries.cockroachdb.com/cockroach-${COCKROACH_VERSION}.linux-amd64.tgz | tar xvz
```

also, ` && rm -f cockroach*.tgz`